.github/workflows/ci.yml

```diff
@@ -145,6 +145,32 @@ jobs:
           gcloud alpha storage cp "${BUILD_DIR_ARCHIVE}" "${GCS_ARTIFACT}"
           echo "::set-output name=gcs-artifact::${GCS_ARTIFACT}"
 
+  build_test_all_bazel:
+    runs-on:
+      # Hacks, and order matters. See the comment at the top of the file.
+      - self-hosted
+      - runner-group=${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}
+      - cpu
+      - os-family=Linux
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2
+        with:
+          submodules: true
+      - name: "Building with Bazel"
+        env:
+          # Don't swap the condition! 0 is falsey and then it would always be 1
+          IREE_BAZEL_WRITE_REMOTE_CACHE: ${{ github.event_name != 'pull_request' && 1 || 0 }}
```

Comment:
I don't quite follow this... why the `&& 1 || 0`? Just to ensure the condition yields either `1` or `0`?

(same below)

Reply:
This is the comment at the top of the file about the hack to create a pseudo-ternary operator. There isn't actually one, but we can use JS-style nonsense to sort of get one. It doesn't work if the first option is falsey though because then it would be `true && 0 [false] || 1 [true]` -> `1` whereas we want it to be 0

Reply:
Ah, but do we actually need a ternary operator for a boolean? The other uses are for strings.

Reply:
We do if we want `1` and `0`, which matches all our other command line booleans. I could force it to coerce to a number some other way, but I don't think it would be less hacky. e.g. I think `github.event_name != 'pull_request' || 0` might also work because it will coerce the data types to match, but I'm not totally sure. Having one hack seems better than two